ci-operator/config/redhat-appstudio/integration-service/redhat-appstudio-integration-service-main.yaml

```diff
@@ -0,0 +1,41 @@
+build_root:
+  image_stream_tag:
+    name: redhat-appstudio-e2e-test-runner
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: Dockerfile
+  to: redhat-appstudio-integration-service-image
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
```

Comment:
probably better to use 4.11 or 4.12 (which should be backward compatible)

Reply:
Thanks. Updated.